.github/workflows/handler.deploy-subgraph.yml

```diff
@@ -1,4 +1,6 @@
 name: Deploy Subgraph
+env:
+  GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
```

Comment:
GITHUB_TOKEN & CI_GITHUB_TOKEN usages are now quite mixed.

There is still places using GITHUB_TOKEN, instead of the globally set CI_GITHUB_TOKEN.

To be clear:

- CI_GITHUB_TOKEN is personal token
- GITHUB_TOKEN is a automatically generated token by github

Reply:
Yes... :disappointed:

Currently, with the automatic `yarn link` through workspaces, we don't actually need this `CI_GITHUB_TOKEN` with the `.npmrc` approach because the build doesn't have to download any @superfluid-finance packages (because they're linked from the monorepo). We could remove all of this `env` code I added...